src/components/simulator/result/SimResultChartAnimation.tsx

```diff
@@ -50,7 +50,7 @@ export const SimResultChartAnimation = ({
   const [displayBalance, setDisplayBalance] = useState(true);
   const gridCols = displayBalance ? 'grid-cols-2' : 'grid-cols-[1fr_80px]';
   return (
-    <>
+    <div data-testid="chart-animation">
```

Comment:
I would rather pass the data-testid to the svg. I think E2E shouldn't impact HTML tree.